template/Dockerfile

```diff
@@ -32,15 +34,27 @@ MAINTAINER Vincent Massol <vincent@massol.net>
 
 # Install LibreOffice + other tools
 # Note that procps is required to get ps which is used by JODConverter to start LibreOffice
+<% if (image == 'debian') print '''\
 RUN apt-get update && \\
   apt-get --no-install-recommends -y install \\
     curl \\
     libreoffice \\
     unzip \\
-    procps \\
-    <% if (db == 'mysql') print 'libmysql-java'
-       if (db == 'postgres') print 'libpostgresql-jdbc-java' %> && \\
-  rm -rf /var/lib/apt/lists/*
+    procps \\''' %>
+    <% if (db == 'mysql' && image == 'debian') print 'libmysql-java && \\'
+       if (db == 'postgres' && image == 'debian') print 'libpostgresql-jdbc-java && \\' %>
+<% if (image == 'debian') print 'rm -rf /var/lib/apt/lists/*' %>
```

Comment:
Is this needed since there's alreayd the IF above about debian?

Reply:
These IF are not nested. The second IF starts after the first IF end